.meta/dietpi-imager

```diff
@@ -246,12 +273,135 @@ Do you want to overwrite or backup the existing file to /root/$OUTPUT_IMG_NAME.$
 			# Rename and move to /root, if source image != output image already
 			[[ /root/$OUTPUT_IMG_NAME.$OUTPUT_IMG_EXT != $FP_SOURCE_IMG ]] && mv "$FP_SOURCE_IMG" $OUTPUT_IMG_NAME.$OUTPUT_IMG_EXT
 
-		# Drive source
-		else
+		# Drive source, clone with dd
+		elif [[ $CLONING_TOOL == 'dd' ]]; then
 
 			G_DIETPI-NOTIFY 2 "Creating final image with actually used size: $(( $IMAGE_SIZE / 1024 / 1024 + 1 )) MiB"
 			dd if=$FP_SOURCE of=$OUTPUT_IMG_NAME.$OUTPUT_IMG_EXT bs=1M status=progress count=$(( $IMAGE_SIZE / 1024 / 1024 + 1 ))
 
+		# Drive source, clone with Clonezilla
+		else
+
+			G_DIETPI-NOTIFY 2 'Creating final image with Clonezilla'
+			# Install required packages
+			G_AG_CHECK_INSTALL_PREREQ clonezilla partclone xorriso isolinux syslinux-common unzip curl || Exit_On_Fail
+			# TODO: 'partimage' is also required by Clonezila, but the package for buster is missing.
+
+			# Download last version of Clonezilla Live
+			CLONEZILLA_VERSION=$(curl -s "$CLONEZILLA_REPO/" | grep 'class="folder "' | head -n1 | cut -d\" -f2)
+			if [[ -z "$CLONEZILLA_VERSION" ]]; then
+
+				G_DIETPI-NOTIFY 1 'Could not retrieve version number of last Clonezilla Live, aborting...'
+				Exit_On_Fail
+
+			fi
+			CLONEZILLA_ZIP="clonezilla-live-${CLONEZILLA_VERSION}-amd64.zip"
+			if [[ ! -f "$CLONEZILLA_ZIP" ]]; then
+
+				G_DIETPI-NOTIFY 2 "Downloading Clonezilla Live v$CLONEZILLA_VERSION..."
+				wget "$CLONEZILLA_REPO/${CLONEZILLA_VERSION}/${CLONEZILLA_ZIP}/download" -O "$CLONEZILLA_ZIP" || Exit_On_Fail
+
+			fi
+
+			# Extract Clonezilla Live
+			unzip "$CLONEZILLA_ZIP" -d 'tmpiso' || Exit_On_Fail
+
+			# Clone disk with Clonezilla
+			if [[ ! -d tmpiso/home/partimag/$OUTPUT_IMG_NAME ]]; then
+
+				mkdir -p tmpiso/home/partimag/
+				G_DIETPI-NOTIFY 2 'Cloning disk with Clonezilla...'
+				/usr/sbin/ocs-sr -or /root/tmpiso/home/partimag --nogui -fsck -q2 -c -j2 -z1p -i 4096 -senc -sc savedisk $OUTPUT_IMG_NAME ${FP_SOURCE##*/} || Exit_On_Fail
```

Comment:
No need to use full binary path here since its inside the $PATH, however does not hurt as well.

Reply:
Changed.